freeipa/src/main/java/com/sequenceiq/freeipa/kerberosmgmt/v1/AuthorizationUtil.java

```diff
@@ -0,0 +1,34 @@
+package com.sequenceiq.freeipa.kerberosmgmt.v1;
+
+import com.sequenceiq.cloudbreak.auth.altus.Crn;
+import com.sequenceiq.cloudbreak.auth.altus.GrpcUmsClient;
+import com.sequenceiq.cloudbreak.logger.MDCUtils;
+import org.springframework.security.access.AccessDeniedException;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+
+@Component
+public class AuthorizationUtil {
+
+    @Inject
+    private GrpcUmsClient grpcUmsClient;
+
+    /**
+     * An authorization check that verifies that a calling actor can perform an operation against a
```

Comment:
I would rephrase a bit saying that the check checks that the calling actor is either perform an action against themselves or have the right.